testframes/test_mne_anonymize/test_mne_anonymize.pro

```diff
@@ -42,7 +42,6 @@ QT += testlib concurrent network
 QT -= gui
 
 CONFIG   += console
-CONFIG   -= app_bundle
```

Comment:
The `CONFIG   -= app_bundle` flag needs to be set. Tests are not supposed to be .app bundles. This should fix the current build.

Reply:
aham .... ok. I'm so used to mess with the flag to be able to debug on mac that I forgot about it.